transaction/atomic.go

```diff
@@ -177,3 +180,62 @@ func AtomicTransfer(hezClient client.HermezClient, ethereumChainID int,
 
 	return
 }
+
+// AtomicTransferJSON receives an array of AtomicTxItems in JSON format, create PoolL2Txs, atomic group, sign and post
+func AtomicTransferJSON(hezClient client.HermezClient, ethereumChainID int,
```

Comment:
Besides server response we should return the atomic group id to help the user to identify the request

Reply:
```func AtomicTransferJSON(hezClient client.HermezClient, ethereumChainID int, txsJSON []string) (serverResponse string, atomicGroupID [32]byte, err error) {```